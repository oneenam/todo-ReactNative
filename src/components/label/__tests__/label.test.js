/**
 * Created by Muhammad Enamul Huq Sarkar on 4/3/18
 */
import React from 'react';
import Label from '../index';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<Label />).toJSON();
    expect(tree).toMatchSnapshot();
});