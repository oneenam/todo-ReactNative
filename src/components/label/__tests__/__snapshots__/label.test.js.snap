// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "verdana",
        "fontSize": 30,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
>
  Text title
</Text>
`;
